packages/portal-proto/src/components/Table/test/ColumnOrdering.unit.test.tsx

```diff
@@ -65,7 +65,7 @@ describe("ColumnOrdering", () => {
       />,
     );
     const button = screen.getByTestId("button-column-selector-box");
-    await userEvent.click(button);
+    await userEvent.click(button, { delay: 100 });
```

Comment:
We might need in line 73 also?

Reply:
Actually, I found `await userEvent.click(button)` in more places too. I am not sure what changed in mantine 7 to fail these tests. Do you have any idea?

Reply:
I do. For dropdowns, the Popup Modal is not part of the DOM unit button is pressed, so the delay seems to allow the DOM to update. Since only this test is failing, maybe just fix it where it's failing?

Reply:
Yes makes sense.